protobuf-maven-plugin/src/main/java/io/github/ascopes/protobufmavenplugin/protoc/ProtocArgumentFileBuilderBuilder.java

```diff
@@ -79,6 +81,11 @@ public ProtocArgumentFileBuilderBuilder setFatalWarnings(boolean fatalWarnings)
     return this;
   }
 
+  public ProtocArgumentFileBuilderBuilder setDescriptorFile(File descriptorFile) {
```

Comment:
Would it be possible to use java.nio.file.Path instead of java.io.File here, just for consistency with the other fields?

Reply:
Also, just thinking about this a bit more, you could probably reuse the `targets` code for this, since it is technically an output target.

```java
private static final class ProtoDescriptorTarget implements Target {
  private final Path outputFile;

  private ProtoDescriptorTarget(Path outputFile) {
    this.outputFile = outputFile;
  }

  @Override
  public void addArgsTo(ArgumentFileBuilder argumentFileBuilder) {
    ...
  }
}
```

Reply:
Changed type to Path and used Target interface